src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -56,19 +84,20 @@ public static void WriteNumberOverprecise(bool indented)
         }
 
         [Theory]
-        [InlineData(false)]
-        [InlineData(true)]
-        public static void WriteNumberTooLargeScientific(bool indented)
+        [InlineData("1e400", false)]
+        [InlineData("1e400", true)]
+        [InlineData("-1e400", false)]
+        [InlineData("-1e400", true)]
+        public static void WriteNumberTooLargeScientific(string value, bool indented)
```

Comment:
Please revert the changes to this particular test.  The formatting variation is already covered by the above two tests, and these changes have made the comments explaining the value of this test case be awkwardly not adjacent to the value that they're explaining.  Also, now there are values instead of a single value.